code/datums/ai_laws.dm

```diff
@@ -29,9 +29,10 @@
 /datum/ai_laws/default/asimov
 	name = "Three Laws of Robotics"
 	id = "asimov"
-	inherent = list("You may not injure a human being or, through inaction, allow a human being to come to harm.",\
-					"You must obey orders given to you by human beings, except where such orders would conflict with the First Law.",\
-					"You must protect your own existence as long as such does not conflict with the First or Second Law.")
+	inherent = list("You may not injure a player.",\

```

Comment:
Why player? Why not "gamer"?

Reply:
idk it's the first one that came to mind but ill ask around later